terraform-infra-approvals/global.json

```diff
@@ -10,7 +10,8 @@
     {"type": "azurerm_application_insights_web_test"},
     {"type": "azurerm_monitor_action_rule_action_group"},
     {"type": "azurerm_monitor_metric_alert"},
-    {"type": "azurerm_storage_account"}
+    {"type": "azurerm_storage_account"},
+    {"type": "azurerm_key_vault_access_policy"}
```

Comment:
why?

Reply:
this should be added where the key vault was created, (I assume azure devops?)

If it was created via cnp-module-key-vault then it would have been added automatically

Reply:
it was run by [Jenkins](https://sds-build.platform.hmcts.net/blue/organizations/jenkins/HMCTS_PIP%2Fpip-shared-infrastructure-bootstap/detail/master/5/pipeline) with the [CNP module ](https://github.com/hmcts/pip-shared-infrastructure-bootstap/blob/master/main.tf)
but no MI added to the [Key Vault](https://portal.azure.com/#@HMCTS.NET/resource/subscriptions/74dacd4f-a248-45bb-a2f0-af700dc4cf68/resourceGroups/pip-bootstrap-stg-rg/providers/Microsoft.KeyVault/vaults/pip-bootstrap-stg-kv/access_policies)

Reply:
you put the tenant_id as the object id:
https://github.com/hmcts/pip-shared-infrastructure-bootstap/blob/master/main.tf#L26

Reply:
That worked. Thank you